docs/data/sql_functions.yml

```diff
@@ -658,6 +658,9 @@ collection:
   - sql: MAP_KEYS(map)
     table: MAP.mapKeys()
     description: Returns the keys of the map as array. No order guaranteed.
+  - sql: MAP_UNION(map1, map2)
+    table: map1.mapUnion(map2)
+    description: Returns a map created by merging two maps, 'map1' and 'map2'. These two maps should have same data structure. If there are overlapping keys, the value from 'map2' will overwrite the value from 'map1'. If any of maps is null, return null.
```

Comment:
it's not clear what is same datastructure in SQL context...
i guess same type (or not same, however common)

Reply:
ok